config/plugins.js

```diff
@@ -63,6 +63,23 @@ module.exports = ({ env }) => ({
       customLabels: {
         name: "strapi-prometheus",
       },
-    }
-  }
-});
\ No newline at end of file
+    },
+  },
+  googledrive: {
+    enabled: true,
+    resolve: './src/plugins/googledrive',
+  },
+  upload: {
+    config: {
+      provider: 'googledrive',
+      providerOptions: {
+        publicFiles: true,
+        uniform: false,
+
+        serviceAccount: env.json('SERVICE_ACCOUNT'), // replace `{}` with your serviceAccount JSON object
```

Comment:
Can we use a more specific env var, like `GOOGLE_DRIVE_SERVICE_ACCOUNT`? We might have other service accounts in the future.